src/calendar/calendar-day.style.tsx

```diff
@@ -151,57 +98,39 @@ export const OverflowDisplay = styled.div<OverflowDisplayProps>`
     }}
 
     ${(props) => {
-        const { $hovered, $point, $selected } = props;
+        const { $selected } = props;
 
-        if (!$hovered && $point === "none" && $selected) {
-            return css`
-                border-top: 1px solid ${Color.Accent.Light[4](props)};
-                border-bottom: 1px solid ${Color.Accent.Light[4](props)};
-                background-color: ${Color.Accent.Light[5](props)};
-            `;
-        }
-
-        if (!$hovered && $point === "after-end" && $selected) {
+        if ($selected) {
             return css`
                 border-top: 1px solid ${Color.Accent.Light[4](props)};
                 border-bottom: 1px solid ${Color.Accent.Light[4](props)};
                 background-color: ${Color.Accent.Light[5](props)};
             `;
         }
+    }}
+    
+    ${(props) => {
+        const { $hovered, $overlap, $selected } = props;
 
-        if ($hovered && $point === "start" && $selected) {
-            return css`
-                background-color: ${Color.Accent.Light[4]};
-            `;
-        }
-
-        if ($hovered && $point === "selected" && $selected) {
+        if ($overlap && $selected) {
             return css`
                 border-top: 1px solid ${Color.Accent.Light[4](props)};
                 border-bottom: 1px solid ${Color.Accent.Light[4](props)};
-                background-color: ${Color.Accent.Light[5](props)};
-            `;
-        }
-
-        if ($hovered && $point === "end" && $selected) {
-            return css`
-                background-color: ${Color.Accent.Light[4]};
+                background-color: ${Color.Accent.Light[4](props)};
```

Comment:
btw if using styled-components `css`, you can just do
```
css`
    background-color: ${Color.Accent.Light[4]};
`
```

Reply:
i see. 

when we should use `(props)`?

keith updated that should added `(props)` behind if under `${props => {}}`

Reply:
the `css` helper is usually sufficient as it also passes down props to nested interpolations

otherwise, scenarios are:

1. you're not using `css`, then you have to manually pass down the props
```
styled.div`
  ${props => `color: ${Color.Primary(props)};`
`
```

2. you modify the props for whatever reason
```
${props => {
  const modifiedProps = { ...props, theme: BaseTheme };
  return css`color: ${Color.Primary(modifiedProps)};`
}
```

thanks for highlighting; the docs will be updated

Reply:
wa. thanks sharing and those example!